### ISSUE TITLE
Windows 8: Toolbar partially hidden

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#149]
```
This issue was reported by a user in Colorado.  So far we cannot reproduce it 
on our own Windows 8.1 test machine.  Here are the details from the user:

GeoDa version: 1.4.6 and 1.5.x
OS: Windows 8 64-bit
Computer: new Dell XPS

To do: verify that user is on Windows 8 and not 8.1



```

Original issue reported on code.google.com by `mmcc...@gmail.com` on 10 Jun 2014 at 5:45

Attachments:
* [GeoDa_win_8_toolbar_cut_off.jpg](https://storage.googleapis.com/google-code-attachments/geoda/issue-149/comment-0/GeoDa_win_8_toolbar_cut_off.jpg)

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.89 (Win 7 Professional (64-bit)).
By default, regression icon in toolbar is still cut off.

**Comment by lixun910** [comment]
please verify in 1.7.91

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.91 (Win 7 Professional (64-bit)).
